### DISCUSSION TITLE
Are you interested in bat themes ?

### DISCUSSION THREAD

**Discussion by nicos68** [EdenEast/nightfox.nvim#449] [Ideas]
Hi,
I’m using dayfox as light theme for everything in my terminal, including bat, for which I just made my own dayfox theme. You can find it in my [dotfiles](https://codeberg.org/nise/dotfiles/src/branch/main/.config/bat/themes/dayfox.tmTheme). I basically started off of the [catppuccin bat theme](https://github.com/catppuccin/bat) and converted it to a dayfox theme.
I could also take the time to convert my dayfox theme to all the other flavors and send you a PR. Would you be interested ? Of course I’m open to any modification suggestion.

**Comment by porridgewithraisins** [comment]
Would love a bat theme for nordfox and nightfox

**Comment by nicos68** [comment]
@EdenEast could you please confirm that if I make bat theme files for all flavours you would be willing to merge them ? I would like to avoid diving into this without being sure it’s going to be used.

**Reply by EdenEast** [reply]
I would merge a bat theme as part of `extra`'s. If you are to add a new extra please add it to the existing extra tooling. The entry point for extras is located [misc/extra.lua](https://github.com/EdenEast/nightfox.nvim/blob/main/misc/extra.lua) and you can find examples of creating an extra template under [lua/nightfox/extras](https://github.com/EdenEast/nightfox.nvim/tree/main/lua/nightfox/extra).

To generate the extra files there is a Make target called `extragen`.

**Reply by nicos68** [reply]
It’s ready for review
https://github.com/EdenEast/nightfox.nvim/pull/457?new_mergebox=true
